db/schema.rb

```diff
@@ -58,6 +58,7 @@
     t.datetime "updated_at", precision: nil, null: false
     t.string "state", default: "waiting", null: false
     t.text "error_message"
+    t.jsonb "objects_with_error", default: [], null: false
```

Comment:
i don't see anything in the migration about adding this column... cruft from a prior iteration, or am i missing something?